make/mt-os/mt-cc3200

```diff
@@ -47,8 +50,8 @@ else ifneq (,$(findstring Windows,$(XI_HOST_PLATFORM)))
 else ifeq ($(XI_HOST_PLATFORM),Linux)
 	# linux cross-compilation prerequisite downloads
 
-	XI_CC3200_PATH_CCS_TOOLS ?= $(HOME)/Downloads/xi_artifactory/ti/ccsv6/tools
-	XI_CC3200_PATH_SDK ?= $(HOME)/Downloads/xi_artifactory/ti/CC3200SDK_1.2.0/cc3200-sdk
+	XI_CC3200_PATH_CCS_TOOLS ?= ~/Downloads/xively-client-artifactory/ti/ccsv6/tools
```

Comment:
good point, I've lost reasons why HOME and why ~(tilde).
I've moved all to ~ alread, is it ok with you David, or change all to HOME?